assignments/lambda-classes.js

```diff
@@ -1 +1,124 @@
 // CODE here for your Lambda Classes
+
+//Person, Instructor, Student, PM
+
+// Person
+// name, age, location
+// speak()
+
+class Person{
+    constructor(propsPerson){
+        this.name = propsPerson.name
```

Comment:
make sure you are using semantical code and including your semi colons after each list